storage/replica.go

```diff
@@ -477,6 +483,8 @@ func newReplica(rangeID roachpb.RangeID, store *Store) *Replica {
 		store:      store,
 		abortCache: NewAbortCache(rangeID),
 	}
+	r.raftMu = syncutil.NewTimedMutex(defaultReplicaRaftMuWarnThreshold)
```

Comment:
Actually we can move the `log.WithLogTag(ctx, "r", &r.rangeStr)` here. Feel free to just use `store.Ctx()` and leave a TODO for me.